frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -689,7 +689,10 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                       }
                     : undefined
 
-                const includeRevenue = !!featureFlags[FEATURE_FLAGS.WEB_REVENUE_TRACKING]
+                // the queries don't currently revenue when the conversion goal is an action
+                const includeRevenue =
+                    !!featureFlags[FEATURE_FLAGS.WEB_REVENUE_TRACKING] &&
+                    !(conversionGoal && 'actionId' in conversionGoal)
```

Comment:
That's a confusing statement, I usually would apply De Morgan here, but I think this is fine, it'll look nicer once we remove the FF

```suggestion
                    (!conversionGoal || !('actionId' in conversionGoal))
```

Reply:
Haha I started with this and decided I preferred the other way. I also tried

```ts
(!conversionGoal || ('eventName' in conversionGoal))
```